example/src/api/firstProvider/clientDetails.ts

```diff
@@ -59,13 +59,13 @@ interface ApplicationDetailsDto {
     };
     someUnionType: 'AAA' | 'BBB' | 100;
     stringAliasType: StringAlias;
-    /** @pact-datetime */
+    /** @pact-matcher isoDatetime */
     datetimeString: string;
-    /** @pact-datetime-with-millis */
+    /** @pact-matcher isoDatetime-with-millis */
```

Comment:
Name `isoDatetimeWithMillis` for consistency

Reply:
I've changed all matchers into names separated by '-'
`iso-datetime-with-millis`